test/e2e/metrics/shared/namespace_selector_test.go

```diff
@@ -138,11 +138,18 @@ func TestNamespaceSelector(t *testing.T) {
 			assert.BackendReachable(t, backend2)
 			assert.DeploymentReady(t, kitkyma.MetricGatewayName)
 
-			if suite.ExpectAgent(tc.label) {
-				assert.DaemonSetReady(t, kitkyma.MetricAgentName)
+			// get the content of the configmap used for the metric gateway
+			if suite.DebugObjectsEnabled() {
+				objects := []client.Object{
+					&includePipeline,
+					&excludePipeline,
+					kitk8s.NewConfigMap(kitkyma.MetricGatewayBaseName, kitkyma.SystemNamespaceName).K8sObject(),
+				}
```

Comment:
Do we need it?

Reply:
not in its current state. I wanted to introduce this pattern into our tests and modify the gha to upload the configuration as artifacts during the test runs
It makes debugging a lot easier. obviously it is off by default